srcpkgs/pari-seadata-small/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'pari-seadata-small'
+pkgname=pari-seadata-small
+version=20090618
+revision=1
+wrksrc=data
+depends="pari"
+short_desc="PARI/GP package needed by ellap for large primes up to 350 bits"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://pari.math.u-bordeaux.fr/"
+distfiles="https://pari.math.u-bordeaux.fr/pub/pari/packages/seadata-small.tgz"
+checksum=bf5be913472b268df7f1242f94c68113fcacceb30c280507447ff2be62760a8f
+
+do_install() {
+	vmkdir  /usr/share/pari
+	vcopy seadata/ /usr/share/pari
```

Comment:
judging from the look of it, pari-seadata{,-big,-small} should conflict each other since they provide the same path?

Reply:
That's a nice question , I think that they can coexist , and on the app the "big" only lest called when the numbers get big than X.

I have looked on the source package for arch and debian and didn't found anything saying that they can coexist . 

But if you would like I can add that to the code saying that they conflict . 

How would you like to proceed ?

Reply:
@motoro. please build all modules with `xbps-src -C ....` and inspect `$MASTERDIR/destdir`, you'll see that seadata packages provide a few of the same path

Reply:
Yes, and my suggestion below should resolve it. Those duplicated path is likely the same content, judging from the same hash.

Reply:
true. so just ship one of those and (if really required!) split it into subpackages

Reply:
Applied @sgn patch , I am going to start working on spliting into the subpackages now.

can the subpackages have is own distfiles ? For what I saw from the manual they don't have ... But correct me if wrong !